cmd/kubeadm/app/util/system/logutils.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
if we HAVE to keep this file (I really hope the answer is no) then I don't know where to put it. I stuck it in with the validators.... but in theory anything that gets removed from k/k should use it to return logs? right? what do you all think?